<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InstallUTM</class>
 <widget class="QMainWindow" name="InstallUTM">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>416</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>602</width>
    <height>416</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>602</width>
    <height>416</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>241</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Выберите ключ для первого УТМа</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>130</y>
      <width>241</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Выберите ключ для второго УТМа</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_UTM1">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>321</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_UTM2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>160</y>
      <width>321</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_install">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>350</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Установить</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>210</y>
      <width>241</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Укажите путь до установочного файла УТМ</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_utm">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>240</y>
      <width>321</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>240</y>
      <width>31</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_cancel">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>350</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Отмена</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_install_gif">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>130</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_install">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>511</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Идет установка, пожалуйста, подождите...
Во время установки, пожалуйста не отключайте рутокены и компьютер...
Так же во время установки УТМ может ругаться на проблемы с RSA ключом, это нормально...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_install_info">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>110</y>
      <width>421</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>77</x>
      <y>140</y>
      <width>441</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_cancel_gui">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>350</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Отмена</string>
    </property>
   </widget>
   <zorder>label_install</zorder>
   <zorder>label_install_gif</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>comboBox_UTM1</zorder>
   <zorder>comboBox_UTM2</zorder>
   <zorder>pushButton_install</zorder>
   <zorder>label_3</zorder>
   <zorder>lineEdit_utm</zorder>
   <zorder>toolButton</zorder>
   <zorder>pushButton_cancel</zorder>
   <zorder>label_install_info</zorder>
   <zorder>progressBar</zorder>
   <zorder>pushButton_cancel_gui</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>602</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
